shell/initialize/app-extended.js

```diff
@@ -52,17 +26,17 @@ async function createApp() {
 
   // here we inject the router and store to all child components,
   // making them available everywhere as `this.$router` and `this.$store`.
-  const app = {
+  const appPartials = {
```

Comment:
I could not find a better name for this, since in the end there's not a real object scope, but rather parts of the app to be used.